pkg/scraper/xpath.go

```diff
@@ -25,6 +25,8 @@ import (
 // configurable at some point.
 const scrapeGetTimeout = time.Second * 30
 
+var debugMode = false
```

Comment:
Really want to avoid global variables if possible. I'll refactor this in a later change.

Reply:
Yeah i try to avoid as much as possible  also, but in the specific case it was a lot easier/quicker doing that.